rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/CreateQueueDirectives.scala

```diff
@@ -115,6 +117,21 @@ trait CreateQueueDirectives {
   }
 }
 
+case class CreateQueueRequest(QueueName: String, Attributes: Option[Map[String, String]], tags: Option[Map[String, String]])
+
+object CreateQueueRequest {
+  implicit val format: RootJsonFormat[CreateQueueRequest] = jsonFormat3(CreateQueueRequest.apply)
+
+  implicit val fpr: FlatParamsReader[CreateQueueRequest] = new FlatParamsReader[CreateQueueRequest] {
```

Comment:
Maybe we change a variable name here? It's hard to say what it is without looking at the type + maybe we also change `format` so it's more verbose.

Maybe something like
`requestJsonFormat` or `requestJsonFormatter`
and
`requestParamsReader`
wdyt?